cloud-tasks/app/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "postcard-app",
```

Comment:
We should make this package explicitly private.
https://docs.npmjs.com/files/package.json#private

Reply:
ack.